packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -260,5 +260,9 @@ export const TypingIndicator = (props: TypingIndicatorProps): JSX.Element => {
     styles
   );
 
-  return <Stack className={mergeStyles(typingIndicatorContainerStyle, styles?.root)}>{indicatorComponent}</Stack>;
+  return (
+    <Stack dir="ltr" className={mergeStyles(typingIndicatorContainerStyle, styles?.root)}>
```

Comment:
How come we need to hard code ltr here?

Reply:
Does it flip the string if we don't? (I'd assume rtl to be the same as ltr for this component)

Reply:
Don't we want the layout of the typing indicator to come in from the right if its rtl?